jupyterlab_server/app.py

```diff
@@ -17,6 +17,19 @@ class LabServerApp(ServerApp):
 
     lab_config = LabConfig()
 
+    blacklist_uris = Unicode('', config=True,
```

Comment:
I see that the behavior of adding multiple URLs is basically a concat of all those lists. However, I wonder about just supporting one URL for now, to keep things as simple as possible. What was the thinking around supporting multiple?

Reply:
The goal is for an org to override in a way the default blacklist. At the beginning, we said that jupyterlab would publish a blacklist (that idea may have changed now). An org would then want to urgently blacklist or strip the user, still relying on the community maintained blacklist.

So with multiple uris you can rely on work done by others. You could also in a org have multiple departments that the user belongs to and those departements maintain their own blacklist/whitelist.

Reply:
Sounds good.